samples/contrib/kubeflow-e2e-mnist/kubeflow-e2e-mnist.ipynb

```diff
@@ -9,13 +9,13 @@
     "In this notebook you will create e2e mnist Kubeflow Pipeline to perfom:\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #25.        create_kserve_task(name,namespace,tfjob_op,model_volume_op)
</pre><p>Since function name changed to <code>create_serving_task</code> above, let's call that here. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/pipelines/pull/7355/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>nit: to keep in line with the style, let's add spaces after commas in the argument list.</p>